data/human/human missions.txt

```diff
@@ -6891,3 +6891,370 @@ mission "Returning to Paradise"
+				accept
+
+	on visit
+		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`
+
+
+
+mission "Adelita 4"
+	name "Take the film to <planet>"
+	description "Take the film to <destination>, where the rest of the Adelita group remained behind. (You will be paid <payment> after you complete the entire job.)"
+	landing
+	passengers 1
+	cargo "stealth drones" 1
+	infiltrating
+	"apparent payment" 200000
+	source "Hippocrates"
+	destination "Maker"
+	to offer
+		has "Adelita 3: done"
+
+	on offer
+		conversation
+			branch "player has a fleet"
+				"total ships" - ( "ships: Fighter" + "ships: Drone" ) > 1
+
+			`It's not hard to find a remote canyon within range of the target area. You deactivate your shields but keep the <ship> hovering in case you need to make a quick escape. You are low enough that Bunker can release the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
+				goto "on site"
+
+			label "player has a fleet"
+			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You deactivate your shields but keep the <ship> hovering in case you need to make a quick escape while Bunker releases the drones from the open cargo bay. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
+
+			label "on site"
+			`	As you watch the display, the drones arrive on site, and you can see a lone Syndicate-built freighter with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`
+			`	He hums to himself for a while before he turns to you and starts talking again. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social pressure, that could stop the slave ships forever."`
```

Comment:
```suggestion
			`	He hums to himself for a while before he turns to you and starts talking again. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now. There's just going to be another slave ship tomorrow. But if we create political and social pressure, that could stop the slave ships forever."`
```

Reply:
Not sure about this. I don't think the full stop / period emphasises how liberating the slaves won't do much, compared to the semicolon. It's also two shortish sentences next to each other which to me is a little jarring.

Reply:
Sure, makes sense. I'm good with keeping this as-is then. Feel free to marked resolved once you see this!